features/unit/sourcemap.feature

```diff
@@ -0,0 +1,22 @@
+@unit
+Feature: SourceMap
+
+    @unit.sourcemap.pctoline
+    Scenario Outline: Parse a SourceMap from json and get the line corresponding to a PC
+        Given a source map json file "<sourcemap-file>"
+        Then getting the line that corresponds to the PC "<pc-value>" produces "<line-number>"
+
+        Examples:
+            | sourcemap-file | pc-value | line-number |
```

Comment:
Thanks for making the tests! 

One thought I had was since it's easy to loop and append strings in the SDKs anyway, we could just check against an entire string like: `0:0,1:2,2:2,3:3,4:3,5:3,6:4,7:5,8:6,9:8,10:8,11:9,12:9,13:9,14:10,15:11,16:12`

In the SDK, it could generate the string and compare against it:
```
decoded_map = source_map.SourceMap(sm)
result_string = ''
for k, v in decoded_map.pc_to_line.items():
    result_string += f'{k}:{v},'
print(result_string[:-1])
```

Reply:
good point, I'll implement this